library/common/extensions/filters/http/test_event_tracker/BUILD

```diff
@@ -0,0 +1,45 @@
+load(
+    "@envoy//bazel:envoy_build_system.bzl",
+    "envoy_cc_extension",
+    "envoy_extension_package",
+    "envoy_proto_library",
+)
+
+licenses(["notice"])  # Apache 2
+
+envoy_extension_package()
+
+envoy_proto_library(
+    name = "filter",
+    srcs = ["filter.proto"],
+    deps = [
+        "@envoy_api//envoy/config/common/matcher/v3:pkg",
+    ],
+)
+
+envoy_cc_extension(
+    name = "test_event_tracker_filter_lib",
```

Comment:
im curious about the `test` prefix in the name `test_event_tracker_`...

Reply:
The prefix is because this filter is intended to be used in tests only  I needed to add this test filter since without it there is no place in Envoy Mobile that emits any events and I want to be test that events emissions works as expected. The addition of a filter was suggested in my [previous PR](https://github.com/envoyproxy/envoy-mobile/pull/1590#discussion_r670649498) and I decided to call it `test_...` since we already have `test_accessor` filter. There may be a better way to do this //cc @junr03 who worked on test_accessor filter in case he has opinions.

Reply:
will add some comments that will make it clearer that the filter is suppose to be for testing purposes only.

Reply:
> Overall I think this looks good. What's the story with the Android side of things?

We are going to start working on Android changes after iOS/common changes from this PR land. @jingwei99 offered to help with them :)